src/khoj/routers/helpers.py

```diff
@@ -683,14 +696,14 @@ def generate_chat_response(
     conversation_id: int = None,
     location_data: LocationData = None,
     user_name: Optional[str] = None,
+    image_url: Optional[str] = None,
```

Comment:
If there's an image_url included, let's update the chat model selection logic to pick a vision model for the response and override the user selection.

Reply:
should we just override to gpt-4o-mini for now? @debanjum would like to hear your thoughts on the best default vision model as well

Reply:
I'd override to the first selection of a model that has `vision_enabled`. Eventually we can let the user select their custom vision model configuration.

Reply:
done!